doc/api/stream.md

```diff
@@ -1263,6 +1265,106 @@ implementors should not override this method, but instead implement
 [`readable._destroy`][readable-_destroy].
 The default implementation of `_destroy` for `Transform` also emit `'close'`.
 
+### stream.pipeline(...streams[, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `...streams` {Stream} Two or more streams to pipe between.
+* `callback` {Function} A callback function that takes an optional error
+  argument.
+
+A class method to pipe between streams forwarding errors and properly cleaning
+up and provide a callback when the pipeline is complete.
+
+```js
+const { pipeline } = require('stream');
+const fs = require('fs');
+const zlib = require('zlib');
+
+// Use pipeline api to easily pipe a series of streams
+// together and get notificed the pipeline is fully done.
+// Here we gzip a potentially huge tar file efficiently:
```

Comment:
I am not sure for this one, but our style guide usually discourages using personal pronouns in the API docs)